### ISSUE TITLE
Can this be used to intercept connections to all SSH servers, instead of just the specified one?

### ISSUE THREAD

**Issue by unsocial-bleach** [magisterquis/sshhipot#7]
I want to use this to intercept all SSH connection traffic, and not just the one specified with the `-cs` flag. Is this possible currently?

**Comment by magisterquis** [comment]
Howdy.

I've not touched the code in a good long time, but, no, it was only really meant to connect to a single server.

You basically want to MitM all SSH traffic on a network?

**Comment by unsocial-bleach** [comment]
That's correct yeah, I want to MitM all SSH traffic. [This is a diagram](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&title=VPN-SSH-MITM#R5Vhdc5swEPw1fkzGQsaxHxPiNp1p0jTOtOlTR2MUUC10jBA27q%2BvBOLLxE7ScU2bPplbTiDt7Z5kBtiLsveSxOE1%2BJQPnKGfDfDlwHEQwo7%2BMcimQMauWwCBZL5NqoE5%2B0ktOLRoynyatBIVAFcsboMLEIIuVAsjUsK6nfYIvP3WmAS0A8wXhHfRr8xXYYFOnLMav6IsCO2b3QkubkSkzLULSULiw7oB4dkAexJAFVdR5lFuuCtpKca923G3mpekQr1kQLYUbL36%2FhlFMHtQ93fpzZKejO3c1KZcL%2FX18m0IUoUQgCB8VqMXElLhU%2FPUoY7qnI8AsQaRBn9QpTa2liRVoKFQRdzepRlTD43rb%2BZRp66NLjP75DzYlIFQcvPQDBqjTFgPy6NyXLE%2Bs6idtFkogVQu6B6urJAVkQFVe%2FJGVXG1KShEVM9Hj5OUE8VW7XkQq86gyqsrqC9sEV9RUDvJFeGpfdN8fqUBjzOzYGeIcm0umQhq0zAQua0MC0mYULmiEp0uIBrgc8fpSKQtgHXIFJ3HJOdurZtAu9h2PlQqmu0vQZcyO2BsHWQ7CJraeF37EZUuCxteLMcdnGS3Q3IfNjqgtPGRpJ0PPZeSbBoJMTChksaTbw1Ql9%2BZtOuPtxvdM%2FllXFe8mEFd%2F2opvy8J%2FJzvnBf6zvlrfef07rvyTPB2jDd6ofGmfRjPLY9qpQDc%2Fcbr5KMjGA91d7z%2FRBNoh3uPLIrp60SB8RFEMepo4lNMxZfbGw3O8zZreEqFKNqxbtUn1x%2FurzvCeabHtlVzgI6Lt7euUd8t96zDpFftXVolQyi4rOjdIlBTodqcJUrCknrAQWpEgDD2e2Scb0GEs0DocKGp06%2FAF4ZYpv%2BXndsbEfP93LtPFenglak2v7IyuFuZ8ROFwX%2BqMNNOYZ48urdVbopj5d%2Bz0sdo61A%2F6VvpCO1m1PkHGcXD3hntHorvaJByYjrGvXdrjsfNZvJWe4eLt3pHuSUevnfosP6iVOym9Wc5PPsF) of the situation I'm considering. 

Do you have any advice on how to go about modifying this project to support this sort of thing? Or is it not really possible?
